app/emoji.go

```diff
@@ -39,11 +39,11 @@ const (
 
 func (a *App) CreateEmoji(sessionUserId string, emoji *model.Emoji, multiPartImageData *multipart.Form) (*model.Emoji, *model.AppError) {
 	if !*a.Config().ServiceSettings.EnableCustomEmoji {
-		return nil, model.NewAppError("UploadEmojiImage", "api.emoji.disabled.app_error", nil, "", http.StatusNotImplemented)
+		return nil, model.NewAppError("UploadEmojiImage", "api.emoji.disabled.app_error", nil, "", http.StatusForbidden)
```

Comment:
Should this be consistent with the ones in file.go, where we are returning `StatusBadRequest`?

Reply:
Actually, it should be the other way round. If you see my PR message, what I am doing is missing license or config not correctly set is returned as forbidden. That is the closest status code I could see.

Reply:
Sounds good!